site/features.md

```diff
@@ -55,6 +56,20 @@ now re-use the same tools and techniques when deploying applications
 as containers as we would have done when deploying them 'on metal' in
 our data centre.
 
+### <a name="optimised-traffic-transport"></a>Optimised traffic transport
+
+Weave automatically chooses the fastest available method to transport
+traffic between peers. The most performant of these ('fastdp') offers
+near-native throughput and latency but does not support encryption;
+consequently supplying a password will cause the router to fall back
+to a slower mode ('sleeve') that does. You can see which method is in
```

Comment:
"...does not support encryption. The slower method ('sleeve') is selected when [encryption is enabled](#security), or communication via the 'fastdp' method fails for some other reason. You can see..."